frontend/lib/ui/artefact_page/test_execution_expandable.dart

```diff
@@ -13,43 +13,56 @@ class TestExecutionExpandable extends ConsumerWidget {
 
   final TestExecution testExecution;
 
-  @override
-  Widget build(BuildContext context, WidgetRef ref) {
+  Row getTestExecutionTileTitle(BuildContext context) {
     final ciLink = testExecution.ciLink;
     final c3Link = testExecution.c3Link;
 
+    return Row(
+      children: [
+        if (!testExecution.status.isCompleted) const SizedBox(width: 36.0),
+        testExecution.status.icon,
+        const SizedBox(width: Spacing.level4),
+        Text(
+          testExecution.environment.name,
+          style: Theme.of(context).textTheme.titleLarge,
+        ),
+        const Spacer(),
+        Row(
+          children: [
+            TestExecutionReviewButton(testExecution: testExecution),
+            const SizedBox(width: Spacing.level4),
+            if (ciLink != null)
+              InlineUrlText(
+                url: ciLink,
+                urlText: 'CI',
+              ),
+            const SizedBox(width: Spacing.level3),
+            if (c3Link != null)
+              InlineUrlText(
+                url: c3Link,
+                urlText: 'C3',
+              ),
+          ],
+        ),
+      ],
+    );
```

Comment:
it just occurred to me that there is a row in a row for no apparent reason

Reply:
I guess we have added it at some point to logically separate the left and right components, but I agree there is no apparent reason to have it, I have deleted it.